### PR TITLE
Fix instance method lookup bug. Fix test suite.

### PR DESCRIPTION
**Pull request by dvogel**
The change motivating these changes is the call to `to_sym`. The rest of the changes are to get the test suite running with rspec 3. I'm fairly certain the tests were broken even with rspec 1 :(

### CODE REVIEW

File: lib/sanitized_attributes/sanitized_attribute.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def sanitize(content)
   def define_ar_writer_method(klass)
     this = self
     attr_name = @attr_name
-    if klass.instance_methods.include?("#{attr_name}=")
+    if klass.instance_methods.include?("#{attr_name}=".to_sym)
```
Comment (mboeh) [inline lib/sanitized_attributes/sanitized_attribute.rb]: `instance_methods` returned an array of strings in 1.8 🍤

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```
Comment (shawnacscott) [inline Gemfile]: Should this be run through the new gem service we have internally?

File: lib/sanitized_attributes/sanitized_attribute.rb (hunk 1 above)
Comment (dvogel) [inline lib/sanitized_attributes/sanitized_attribute.rb]: Yeah, I figured it must be a pre-2.0 difference. Thanks for confirming :)

File: Gemfile (hunk 2 above)
Comment (dvogel) [inline Gemfile]: The rubygems proxy is to make deploys more resilient. The Gemfile is ignored during deploys. It's here only to support `bundle exec rspec`.

**Pull request merged**